advocacy_docs/community/contributing/styleguide.mdx

```diff
@@ -441,7 +436,21 @@ Example: The `CREATE TABLE` statement creates a new table in a database.
 
 Code blocks provide executable code samples, marked with an opening and closing set of three backticks (` ``` `). Code blocks can support syntax highlighting if you add the language name immediately after the first line of backticks. Use one returned line before and after a code block for better Markdown readability. For example:
 
-`This is a sample line of text. ``` {% include copy-clipboard.html %} ~~~shell $ go get -u github.com/lib/pq ~~~ ``` This is more sample text.`
+![Sample Code Block](images/samplecodeblock.png)
```

Comment:
We don't need images for this - can provide the input verbatim by surrounding in a fence built from four backtics, e.g.

`````
````

This is a sample line of text. 

```
{% include copy-clipboard.html %}
~~~shell
$ go get -u github.com/lib/pq
~~~
```

This is more sample text.
````
`````

(and yes, I quoted *that* verbatim by fencing with *five* backtics. Gillette can do it with razor blades, so why not backtics?)

Also, recommend showing rendered output in blockquote for ease of scanning.